tests/virt_autotest/guest_upgrade_run.pm

```diff
@@ -52,6 +52,14 @@ sub get_script_run {
     if (get_var("SKIP_GUEST_INSTALL") && is_x86_64) {
         $pre_test_cmd .= " -k $vm_xml_dir";
     }
+
+    my $do_registration = check_var('GUEST_SCC_REGISTER', 'installation') ? "true" : "false";
+    my $registration_server = "http://all-" . get_required_var('BUILD') . ".proxy.scc.suse.de";
```

Comment:
I am thinking of introducing one more settings in job group level for all test suites. SCC_URL=<proxy_scc> is kept, a new HOST_SCC_URL=<proxy_scc>|<empty_value> based on host versions. And HOST_SCC_URL is used in host autoyast profile. then we need not generate this proxy scc again in test scripts like what you do at here. It would be helpful not only for prj2/prj4 tests but also for possible requirements of adding proxy scc for other tests.

Reply:
Could you help make them a list and corresponding usage and comment below ? (including existing ones that relevant)

Reply:
I have not added them. I raise this idea to discuss with you whether it is better. If you agree, they should be like:

virtualization-acceptance-osd.yaml:
```
defaults:
  x86_64:
    machine: 64bit-ipmi
    priority: 30
    settings: 
     ...
     HOST_SCC_URL: %SCC_URL%
     ...
    - gi-guest_sles12sp5-on-host_developing-kvm:
        machine: 64bit-ipmi-large-mem
        settings:
        ...
        HOST_SCC_URL: ''
        ...
```

and use HOST_SCC_URL in host autoyast profile templates, for both dev_host_12.xml.ep and dev_host_15.xml.ep
```
...
<reg_server>{{HOST_SCC_URL}}</reg_server>
...
```

As of `GUEST_SCC_REGISTER`, it is different thing, nothing to do with HOST_SCC_URL.

Reply:
For HOST_SCC_URL, I think it is OK.

Reply:
Ok, I'll make this change.

Reply:
For guest ugprade, how do you want to set scc url ?

1. Use "http://all-" . get_required_var('BUILD') . ".proxy.scc.suse.de"
2. Use get_var('SCC_URL', '')
3. Introduce GUEST_SCC_URL which should be given the value of SCC_URL under normal circumstances

And you also need to set GUEST_SCC_REGISTER=installation for guest upgrade test if guest registered.
@Julie-CAO

Reply:
> For guest ugprade, how do you want to set scc url ?

> 1. Use "http://all-/" . get_required_var('BUILD') . ".proxy.scc.suse.de"
> 2. Use get_var('SCC_URL', '')
> 3. Introduce GUEST_SCC_URL which should be given the value of SCC_URL under normal circumstances

I would prefer to option 2 not to introduce more setting with save value.

> And you also need to set GUEST_SCC_REGISTER=installation for guest upgrade test if guest registered.

No problem, give me the list of prj4 tests which need set GUEST_SCC_REGISTER=installation and I'll add them in test suites or job group.

Reply:
Already changed to use ```get_var('SCC_URL', 'https://scc.suse.com');```. For ```GUEST_SCC_REGISTER=installation```, could @guoxuguang help confirm which prj4 test has registered guest installed ?  Then could @guoxuguang help do some verification runs on s390x ?

Reply:
s390x lpars have not been workable yet. I am afaid we are unable to verify shortly.

Reply:
@guoxuguang Could you review these 3 PRs for guest upgrade tests both on x86 and s390?